builds/e2e/connectivity.yaml

```diff
@@ -11,7 +11,33 @@ jobs:
 ################################################################################
     displayName: Linux AMD64 Moby
     condition: and(eq(variables['run.linux.amd64.moby'], 'true'), ne(variables['agent.group'], ''))
-    timeoutInMinutes: 120
+    timeoutInMinutes: 180
+    strategy:
+      matrix:
```

Comment:
Is there a way to decide if we want to use matrix strategy or not?
Say... I want to only test the `Offline(5/20/3)` case without going through the rest of the cases.

Reply:
I added a pipeline variable “config.to.run”; you can specify a number between 1-6 to run a specific config.